_dashboards/visualize/maps.md

```diff
@@ -120,6 +120,58 @@ To show a subset of the data in the index, filter the data. The following exampl
 For large datasets, you may want to avoid loading data for the whole map. To load data only for the part of the map that is currently visible, select the **Only request data around map extent** checkbox.
 {: .tip}
 
+### Drawing shapes to filter data
+
+You can filter your data by drawing shapes on your map. You can also modify the filter and remove it completely. In future releases, you will be able to modify the filter shape directly on the map from the Dashboards panel.
+
+When drawing a shape on the map across the International Dateline, you must draw it in a counterclockwise direction. Drawing in a clockwise direction will not include the correct datapoints.
```

Comment:
i think we don't have to call out International Dateline, we can ask use to draw counter clockwise for accuracy.

Reply:
@VijayanB Do we have a link to the GeoJSON specification? It may help to include it.

Reply:
https://www.rfc-editor.org/rfc/rfc7946